javascript/webhooks-nodejs/README.md

```diff
@@ -21,25 +21,36 @@ npm install
 ```
 
-ngrok is used to create a publicly-accessible "tunnel" (i.e., URL) to a port on your local machine. Note that by default, the Express server in this demo will run on port `8080`.
+Ngrok is used to create a publicly accessible "tunnel" (i.e., URL) to a port on your local machine. By default, the Express server in this demo runs on port `8080`, so ensure that ngrok is set up to tunnel to the correct port where your server is running.
 
 ## Usage
 
-1. Start the server:
+1. Create a file named **.env** in the root directory of this app. In the file, include the following line:
+
+```
+X_HOOK_SECRET=
+```
+
+The Express server writes the `X-Hook-Secret` obtained from the [webhook handshake](https://developers.asana.com/docs/webhooks-guide#the-webhook-handshake) directly into this file, allowing the `X-Hook-Secret` to persist locally across server restarts. In a production environment, this value should be securely stored in a database. For security reasons, ensure that you never commit or expose this value in a public environment.
+
+For your convenience, a sample **.env.template** file is included in the root directory of this application.
+
+2. Start the webhook server:
 
 ```
 npm run dev
 ```
 
-2. Create a webhook by calling [POST /webhooks](https://developers.asana.com/docs/establish-a-webhook) and specifying given resource (e.g., a task). Feel free to use the [Postman Collection](https://developers.asana.com/docs/using-postman) to make your request.
+1. Create a webhook by calling `POST /webhooks` and specifying given resource (e.g., a task). Feel free to use the [Postman Collection](https://developers.asana.com/docs/using-postman) or the [POST /webhooks](https://developers.asana.com/reference/createwebhook) page in the API Explorer to make your request.
+
+> **Note:** In your request, make sure to set the [`target`](https://developers.asana.com/docs/webhook) parameter to your public ngrok domain instead of the example `localhost` domain, `http://localhost:8000/receiveWebhook`. This means you must replace `localhost:8000` with your unique ngrok domain. The `target` should be your ngrok server's "Forwarding" domain followed by `/receiveWebhook`. 
+> 
+> The final value for `target` will look something like this: `https://0d32-71-236-53-92.ngrok-free.app/receiveWebhook`.
 
-    * Note:  You must point the [target](https://developers.asana.com/docs/webhook) request parameter to your public ngrok domain instead of the example local host domain `http://localhost:8000/receiveWebhook` (i.e., replace `localhost:8000` with your unique ngrok domain). 
-    * The target will be your ngrok server's "Forwarding" domain appended with `/receiveWebhook` and will look something like:  `https://0d32-71-236-53-92.ngrok-free.app/receiveWebhook`).
 
-3. In the Asana UI (or via the API), update the resource (e.g., change the task name).
+1. In the Asana UI (or via the API), update the resource (e.g., change the task name).
```

Comment:
This numbering seems off unless 1 starting under 2 section as intentional
![numbering off](https://github.com/user-attachments/assets/5955d810-eb4d-42de-bed9-ddcff4de7fe0)